docs/developers/mobile.md

```diff
@@ -0,0 +1,34 @@
+# Mobile Development
+
+## Testing the client on a mobile device
+
+If you have made changes to the client that could affect the mobile experience, it is a good idea to test them on a real device. Such changes should ideally be tested with at least current versions of iOS Safari and Chrome for Android.
+
+1. Make sure your development system and mobile device are on the same local network.
+1. Get the IP address or host name of your development system (`$HOSTNAME` in the steps below). You can do this using the `hostname` terminal command on Mac/Linux.
```

Comment:
> Regarding $HOSTNAME, I take your point but I think setting an env var might give the misleading impression that the commands make use of this env var name internally, as opposed to it just being a placeholder in the docs.

That's a good point. I'd suggest maybe using a style other than `$HOSTNAME`, maybe `<HOSTNAME>`? We could then use the same style throughout for docs placeholders.